release.json

```diff
@@ -10,22 +10,11 @@
 		"OMNIBUS_RUBY_BRANCH": "datadog-5.5.0",
 		"OMNIBUS_SOFTWARE_BRANCH": "master"
 	},
-	"5.28.1-rc.1": {
-		"AGENT_VERSION": "5.28.1-rc.1",
-		"AGENT_BRANCH": "5.28.1-rc.1",
-		"AGENT6_BRANCH": "6.6.0",
-		"INTEGRATIONS_CORE_BRANCH": "5.28.1-rc.1",
-		"TRACE_AGENT_BRANCH": "6.6.0",
-		"TRACE_AGENT_ADD_BUILD_VARS": "true",
-		"PROCESS_AGENT_BRANCH": "6.6.0",
-		"OMNIBUS_RUBY_BRANCH": "datadog-5.5.0",
-		"OMNIBUS_SOFTWARE_BRANCH": "6.6.0"
-	},
 	"5.28.0": {
```

Comment:
You should edit `5.28.1-rc.1` or add a new `5.28.1` instead of reusing `5.28.0`.